extensions/azurePublish/src/components/azureProvisionDialog.tsx

```diff
@@ -96,12 +96,12 @@ const PageTypes = {
 
 const DialogTitle = {
   CHOOSE_ACTION: {
-    title: formatMessage('Configure resources'),
+    title: formatMessage('Configure resources to your publishing profile'),
```

Comment:
While we're making grammatical changes, this phrasing seems weird. Maybe "Configure resources in your publishing profile"? Or "within"? "Using"? I'm not 100% clear on what the relationship is, but "configure to" doesn't sound right in any case.

Reply:
I will run this by design and change accordingly

Reply:
Design has a task all up to review copy and will resolve changes in a seperate PR, I have provided them your input for this one. For this PR we will leave it as is in anticipation of a full PR around copy